src/BK2K/Sitepackage/GeneratorBundle/Resources/skeletons/BaseExtension/fluid_styled_content/ext_localconf.php.twig

```diff
@@ -5,3 +5,7 @@ defined('TYPO3_MODE') || die();
  * Add default RTE configuration
  */
 $GLOBALS['TYPO3_CONF_VARS']['RTE']['Presets']['{{ package.extensionKey }}'] = 'EXT:{{ package.extensionKey }}/Configuration/RTE/Default.yaml';
+
+\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
+    '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:' . $_EXTKEY . '/Configuration/PageTS/">'
```

Comment:
Should be `'<INCLUDE_TYPOSCRIPT: source="FILE:EXT:' . $_EXTKEY . '/Configuration/PageTS/All.txt">'`